controller/users.go

```diff
@@ -128,13 +129,23 @@ func (c *UsersController) Update(ctx *app.UpdateUsersContext) error {
 
 		// Disabling updation of email till we figure out how to do the same in Keycloak Error-free.
 		//
-		/*
-			updatedEmail := ctx.Payload.Data.Attributes.Email
-			if updatedEmail != nil {
-				user.Email = *updatedEmail
-				keycloakUserProfile.Email = updatedEmail
+
+		updatedEmail := ctx.Payload.Data.Attributes.Email
```

Comment:
@sbose78 shouldn't we check if the email has changed to do the thing?
something like ..`if updatedEmail != nil  and user.Email != *updatedEmail`